.github/workflows/sanitizers/toolchain64.cmake

```diff
@@ -0,0 +1,35 @@
+# Configure compiler and ABI flags.
+# This is a cmake toolchain file, designed to be specified by
+# setting the CMAKE_TOOLCHAIN_FILE variable either as a command line parameter
+# by using -D or by calling "set" before the top-level CMakeLists.txt call to
+# project(). **This file should NOT be included directly**.  The purpose
+# of the toolchain file is to prepare the build directory for cross-compiling.
+
+cmake_minimum_required (VERSION 3.22)
+
+set(IS_64BIT yes CACHE BOOL "Tool chain bitness.")
+
+# Set the compiler flags. Use the CACHE instead of the *_INIT variants
+# which are modified in the in the compiler initialization modules.
+set(CMAKE_C_FLAGS "-march=westmere -m64 -fno-strict-aliasing" CACHE STRING "ABI C flags.")
+set(CMAKE_C_FLAGS_RELWITHDEBINFO "-g -O2 -fno-omit-frame-pointer" CACHE STRING "ABI C flags.")
```

Comment:
If we build everything in Debug, why do we specify this setting for "release with debug info"?

Reply:
Removed.